spec/support/shared/todo/newly_created_todo.rb

```diff
@@ -0,0 +1,9 @@
+shared_examples_for "a newly created todo" do
+  it "should associate the todo with the todo list" do
+    expect(@todo.model.list).to eq(list)
```

Comment:
The `@todo`s here expects @todo to be defined and it's not visible from this context. This creates a dependency to the file that includes this. Can we dry up the spec in another way?